games-engines/openscenegraph/openscenegraph-3.6.5.recipe

```diff
@@ -110,6 +109,7 @@ REQUIRES_devel="
 
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
+	openexr3.2${secondaryArchSuffix}_devel
```

Comment:
Why not use the library names instead of the package names?

Reply:
Easier to maintain? :)

Reply:
Not acceptable, you have been committing long enough (and thanks for that) to know when to include base packages instead of using the library names, in this case there is no reason not to use the library names.